src/Workspaces/Core/Portable/Rename/LightweightRenameLocations.cs

```diff
@@ -91,15 +87,14 @@ internal sealed partial class LightweightRenameLocations
                     {
                         var result = await client.TryInvokeAsync<IRemoteRenamerService, SerializableRenameLocations?>(
                             solution,
-                            (service, solutionInfo, callbackId, cancellationToken) => service.FindRenameLocationsAsync(solutionInfo, callbackId, serializedSymbol, options, cancellationToken),
-                            callbackTarget: new RemoteOptionsProvider<CodeCleanupOptions>(solution.Services, fallbackOptions),
+                            (service, solutionInfo, cancellationToken) => service.FindRenameLocationsAsync(solutionInfo, serializedSymbol, options, cancellationToken),
```

Comment:
we can now call to the server to find the rename locations without needing a callback path for it.